dev-haskell/cabal/cabal-3.2.1.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+#hackport: flags: -bundled-binary-generic
+
+# Drop test-suite due to "circular" dependencies with quickcheck
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+MY_PN="Cabal"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A framework for packaging Haskell software"
+HOMEPAGE="https://www.haskell.org/cabal/"
+SRC_URI="https://hackage.haskell.org/package/${MY_P}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+# dropped ppc and ppc63 due to nonsolvable deps from pkgcheck
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x86-solaris"
+IUSE=""
+
+RDEPEND=">=dev-haskell/fail-4.9:=[profile?] <dev-haskell/fail-4.10:=[profile?]
+	>=dev-haskell/mtl-2.1:=[profile?] <dev-haskell/mtl-2.3:=[profile?]
+	>=dev-haskell/parsec-3.1.13.0:=[profile?] <dev-haskell/parsec-3.2:=[profile?]
+	>=dev-haskell/semigroups-0.18.3:=[profile?] <dev-haskell/semigroups-0.20:=[profile?]
+	>=dev-haskell/text-1.2.3.0:=[profile?] <dev-haskell/text-1.3:=[profile?]
+	>=dev-lang/ghc-7.8.2:=
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+# TODO: Fill this out once the final decision has been made as to which version
```

Comment:
If you don't know if just don't set it. `haskell-cabal.eclass` has the code tom complain at build time when collision happens. It's easy to add afterwards.

Reply:
Perfect, I'll leave it out here and any time I'm unsure.